tgui/packages/tgui/interfaces/Cargo/CargoCart.tsx

```diff
@@ -26,8 +26,8 @@ export function CargoCart(props) {
           <CheckoutItems />
         </Section>
       </Stack.Item>
-      <Stack.Item>
-        {cart.length > 0 && !requestonly && (
+      {cart.length > 0 && !!can_send && (
```

Comment:
```suggestion
      {cart.length > 0 && can_send && (
```
No reason to use double negatives

Reply:
it is used to convert our numeral byond "bools" (zeroes and ones) to actual js bools. Without double negation, it will leave ugly zero at the bottom of the UI.

like here
<details>

![1b59fcbe-e46c-4cab-9277-2c54fac2bf1d](https://github.com/tgstation/tgstation/assets/8430839/b0c8dd0d-b3d7-4601-a31d-cee0190ac318)
 </details>